src/main/java/onlineservices/handlers/MqttClientHandler.java

```diff
@@ -0,0 +1,36 @@
+package onlineservices.handlers;
+
+import org.eclipse.paho.client.mqttv3.MqttCallback;
+import org.eclipse.paho.client.mqttv3.MqttClient;
+import org.eclipse.paho.client.mqttv3.MqttException;
+
+public class MqttClientHandler {
+
+    MqttClient mqttClient;
+
+    public MqttClientHandler(String brokerUrl, String clientId) {
+        try {
+            mqttClient = new MqttClient(brokerUrl, clientId);
+        } catch (MqttException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public void connect() throws MqttException {
+        mqttClient.connect();
```

Comment:
add logging to all methods, ex: connecting mqtt client, connected mqtt client, subscribing to topic, subscribed to topic...